tests/main/views/test_buyers.py

```diff
@@ -2345,7 +2425,7 @@ def test_404_if_brief_has_wrong_lot(self):
         assert res.status_code == 404
 
 
-class TestViewBriefResponsesPageForLegacyBrief(AbstractViewBriefResponsesPage):
+class TestViewBriefResponsesPageForLegacyBrief(BaseApplicationTest):
```

Comment:
The `AbstractViewBriefResponsesPage ` base class contains shared tests that are run for this legacy class and the main one - while the users can no longer create legacy briefs, they can still view their old ones, so we need coverage for this.

Reply:
Does this mean we need to run tests against both frameworks?

Reply:
No, only these ones in the legacy flow need DOS1 - the other journeys should be the same for all iterations.

Reply:
So for the legacy class we test against DOS1, and the main class we test against DOS4?

Reply:
That should work, yes - if we setup `self.framework_slug = ` for each class, then we can refer to that instead of hardcoding the framework on the shared tests and the class-specific tests.

Reply:
Done